hydra-node/test/Hydra/Chain/Direct/StateSpec.hs

```diff
@@ -398,7 +398,7 @@ forAllFanout action =
        in action utxo tx
             & label ("Fanout size: " <> prettyLength (countAssets $ txOuts' tx))
  where
-  maxSupported = 30
+  maxSupported = 39
```

Comment:
Benchmarks show `45`, but there might be some delta between the generators used in `tx-cost` and these tests here. It's fine to only bump it to 39, although we could try higher numbers to test the limits more often.